_topic_maps/_topic_map.yml

```diff
@@ -2229,6 +2229,14 @@ Topics:
   - Name: Placing pods on specific nodes using node selectors
     File: nodes-pods-node-selectors
     Distros: openshift-enterprise,openshift-origin
+  Name: Using run-once pods
+  Distros: openshift-enterprise
+    Dir: run_once_duration-override
```

Comment:
This directory name isn't correct (though yours actually has an .adoc extension in the directory name, so that will have to be removed).

I pulled down and played with it a bit, and the update should look something like this (after you fix the directory name):

```
  - Name: Using run-once pods
    Dir: run_once_duration_override
    Distros: openshift-enterprise
    Topics:
    - Name: Run once duration override operator release notes
      File: run-once-duration-override-release-notes
    - Name: Configuring run once duration override operator
      File: run-once-duration-override-configuring
```
Not the indention of 2 spaces. That puts it under the nodes/pods directory instead of at the same level as it